icq-relayer/pkg/types/config.go

```
@@ -94,3 +113,82 @@ func DefaultReadOnlyChainConfig(chainID string, rpcUrl string) *ReadOnlyChainCon
 		QueryRetryDelayMilliseconds: 400,
 	}
 }
+
+func (c *Config) FilterHA() {
+	seed := 0
+
+	var result map[int][]string
+	chains := slices.Collect(maps.Keys(c.Chains))
+	sort.Strings(chains)
+	for true {
+		seed += 1
```

golangci-lint (1.62.2)

[warning] increment-decrement: should replace seed += 1 with seed++

(revive)